src/hangman/game.clj

```diff
@@ -8,26 +8,28 @@
 
 (def game-id (atom nil))
 
+(defn valid-user-input? [user-input]
+  (let [valid? (= (count user-input) 1)]
```

Comment:
I think the "?" style is only used for functions, not for boolean variables..

You can also use when-not instead of when there